eng/versioning/version_client.txt

```diff
@@ -130,6 +130,8 @@ com.microsoft.azure:spring-cloud-starter-azure-appconfiguration-config;1.2.8-bet
 # Format;
 # unreleased_<groupId>:<artifactId>;dependency-version
 # note: The unreleased dependencies will not be manipulated with the automatic PR creation code.
+unreleased_com.azure:azure-core;1.9.0-beta.1
```

Comment:
There's already an entry for azure-core. This line can be removed.

Reply:
Removed